accounting/receivables/customer_payments/batch_sdd.rst

```diff
@@ -1,117 +1,146 @@
-until it is *Closed* or *Revoked*.
 
-.. image:: media/batch_sdd03.png
+Direct Debit mandates are closed automatically after their *End Date*.  If this field is left blank,
+then the mandate keeps being *Active* until it is *Closed* or *Revoked*.
+
+.. image:: media/batch-sdd-revoke-close.png
    :align: center
+   :alt: Close or Revoke an SDD Mandate in Odoo Accounting
 
-Clicking on **Close** updates the mandate’s end day to the current day. 
-This means that invoices made after the present day won’t be processed 
-automatically with an SDD payment.
+Clicking on **Close** updates the mandate’s end day to the current day. This means that invoices
+made after the present day won’t be processed with an SDD payment.
 
-Clicking on **Revoke** disables the mandate immediately. No SDD 
-payment can be registered anymore, regardless of the invoice’s date. 
-However, payments that have already been registered will still be 
-included to the next SDD XML file.
+Clicking on **Revoke** disables the mandate immediately. No SDD payment can be registered anymore,
+regardless of the invoice’s date. However, payments that have already been registered are still
+included in the next SDD XML file.
 
-.. note::
-   Once a mandate has been *closed* or *revoked*, it cannot be reactivated. 
-   If a customer requires to pay with SDD payments again, you will have 
-   to create a new mandate from scratch.
+.. warning::
+   Once a mandate has been *closed* or *revoked*, it cannot be reactivated.
 
 Get paid with SDD Batch Payments
 ================================
+
 Customer Invoices 
 -----------------
-All new posted invoice issued to customers with an active mandate 
-will automatically register payment and have their status marked as *Paid*.
 
-.. tip:: 
-   If you have unpaid invoices that could be paid with a new mandate, 
-   it is still possible to do it. Go on the invoice, click on *Register Payment* 
-   and choose *SEPA Direct Debit* as payment method.
+You can register SDD payments for invoices issued to customers who have an active SDD mandate.
+
+To do so, open the invoice, click on *Register Payment*, and choose *SEPA Direct Debit* as payment
+method.
+
+.. note::
+   In previous versions, Odoo used SDD payment as the default payment for all customers with an
+   active mandate. Your customers can still activate automatic SDD for their subscriptions with a
+   recurring payment.
 
 Generate SEPA Direct Debit XML files to submit payments
 -------------------------------------------------------
-To submit your SDD payments to your bank, you will first generate an 
-**XML file** that can be uploaded directly to your bank interface.
 
-The files generated by Odoo follow the SEPA Direct Debit **PAIN.008.001.02** 
-specifications as required by the SEPA customer-to-bank Implementation Guidelines, 
-which ensures compatibility with the banks.
+**XML files** with all SDD payment instructions can be uploaded to your bank institution to process
```

Comment:
```suggestion
**XML files** with all SDD payment instructions can be uploaded to your bank account to process
```

Reply:
it's not really uploaded to an account. I'll find another word. Thanks though :)